### PR TITLE
Add 'Dropdowns' page

### PR DESCRIPTION
**Pull request by janfaracik**
Screenshot 2024-02-10 at 13 18 51

Adds a page on dropdowns. 

### Testing done

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@ THE SOFTWARE.
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/design-library-plugin</gitHubRepo>
-        <jenkins.version>2.426.1</jenkins.version>
+        <jenkins.version>2.445-rc34653.8b_944e9e3fe9</jenkins.version>
```
Comment (janfaracik) [inline pom.xml:47]: TODO - Revert before merge

**Review by timja — APPROVED** [review]

File: src/main/resources/io/jenkins/plugins/designlibrary/Dropdowns/index.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description.1=Dropdowns allow you to group similar controls under one roof. They''re an effective way to de-clutter your page whilst offering users the actions they need. 
+description.2=The overflow button displays a temporary menu of actions that relate to one another.
+do=Do
+dont=Don''t
+do.1=Use dropdowns to avoid overcomplicating your page
+dont.1=Avoid moving important, commonly used actions to dropdowns
+do.2=Use dropdowns to offer quick access to a subpage's actions
```
Comment (daniel-beck) [inline src/main/resources/io/jenkins/plugins/designlibrary/Dropdowns/index.properties:7]:
```suggestion
do.2=Use dropdowns to offer quick access to a subpage''s actions
```

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Lacks `dd:custom` example.

**Review by daniel-beck — APPROVED** [review]
Looks reasonable. Untested.

**Pull request merged**